### ISSUE TITLE
Update supported browsers

### ISSUE THREAD

**Issue by nrenner** [nrenner/brouter-web#628]
Currently supported browser are defined by this [Browserslist](https://github.com/browserslist/browserslist) [query](https://browsersl.ist/?q=%3E+0.5%25%2C+last+2+versions%2C+Firefox+ESR%2C+Firefox+%3E%3D+56%2C+not+dead%2C+Explorer+%3E%3D+10%2C+Android+%3E%3D+4.1%2C+Safari+%3E%3D+7%2C+iOS+%3E%3D+7) which is used by Babel, ESLint, Autoprefixer and PostCSS:

https://github.com/nrenner/brouter-web/blob/4c2e39066347fe3c1eecb0b9fb17122f01eec458/package.json#L24-L34

The query is composed of:
- `> 0.5%, last 2 versions, Firefox ESR, not dead` - the Browserslist [`defaults`](https://github.com/browserslist/browserslist#full-list) query
- `Firefox >= 56` - still in use because of support for old plugins before Web Extensions ([#571](https://github.com/nrenner/brouter-web/issues/571))
- `Explorer >= 10, Android >= 4.1, Safari >= 7, iOS >= 7` - minimal versions to support Flexbox ([source](https://davidwalsh.name/goodbye-vendor-prefixes#codekit))

Quoting @tbsmark86 from #621:
> Safari 7, op_mini all, IE 10, android 4.1 don't seem relevant today. Those are all EOL maybe eslint configuration should be updated?

The versions `Explorer >= 10, Android >= 4.1, Safari >= 7, iOS >= 7` were the minimal required when we introduced Flexbox and Autoprefixer. That part could probably be dropped nowadays.

> And supporting opera-mini seems pointless.

The Browserslist definition does not consider it [dead](https://browsersl.ist/?q=dead) yet, still covered by the [defaults](https://browsersl.ist/?q=defaults) (Opera Mini all: 0.95 %).

It would be nice to get [custom usage data](https://github.com/browserslist/browserslist#custom-usage-data) from User-Agent strings in Nginx log, but there only seem to be [tools](https://github.com/browserslist/browserslist#get-statistics-for-5-in-my-stats) for Google Analytics & co.

**Comment by tbsmark86** [comment]
>The Browserslist definition does not consider it [dead](https://browsersl.ist/?q=dead) yet, still covered by the [defaults](https://browsersl.ist/?q=defaults) (Opera Mini all: 0.95 %).

I agree. Opera Mini is not dead. What I meant was that Opera Mini has very limited supported for interactive websites because, as far as I now, most of the rendering is done on the server. 

> The versions Explorer >= 10,

Not sure if you know but IE 11 has reached EOL a few month ago. So drop IE.

>  Android >= 4.1, 

Hm not sure at which to jump but starting 2024 Android < 7.1 will fail because of Lets-Encrypt.
Probably let the browserlist default decide.

> Safari >= 7, iOS >= 7

Not sure about safari but iOS can be at least >= 9. Because apple does update so well an ios 8 device would be more then 10 years old.
